kqueen/blueprints/api/test_organization.py

```diff
@@ -36,6 +36,13 @@ def test_crud_delete(self):
             assert response.status_code == 500
             assert isinstance(remaining, list) and remaining
 
+    def get_create_data(self):
```

Comment:
Unclear naming. Did you mean "creation"?

Reply:
that name defined in base class